aegis-web/yo/Gruntfile.js

```diff
@@ -115,7 +115,7 @@ module.exports = function (grunt) {
           livereload: '<%= connect.options.livereload %>'
         },
         files: [
-          '<%= yeoman.app %>/{,*/}*.html',
+          '<%= yeoman.app %>/**/*.html',
```

Comment:
Why is that change needed?

Reply:
So that the browser will reload the page once any HTML file is saved. Without this change the page has to be reloaded manually to see changes.

Reply:
I was also curious, according to [globbing patterns](https://gruntjs.com/configuring-tasks#globbing-patterns), it looks correct. I think `{,*/}` was a poor implementation of `**`